gallery/src/components/NxList/NxListPage.tsx

```diff
@@ -0,0 +1,290 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxP, NxCode, NxTextLink, NxTable, NxH3, NxInfoAlert, NxTile } from '@sonatype/react-shared-components';
+import {GalleryDescriptionTile} from '../../gallery-components/GalleryTiles';
+import NxListsExamples from './NxListExamples';
+
+const NxListPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        <NxCode>NxList</NxCode> encapsulates the styles defined in the <NxCode>nx-list</NxCode> style page into
+        simple React components. In addition, <NxCode>NxList</NxCode> also contains children components that make
+        it easy to customize lists. With the help of these components, it is quite simple to create lists that have
+        clickable buttons, links, action buttons, multiple lines of text, and even lists with empty and error states.
+      </NxP>
+
+      <NxH3>Description Lists</NxH3>
```

Comment:
Should be a subsection

Reply:
> Should be a subsection

9d30cb238f7426de9c9aed006407ac9ac7634fd9